.github/workflows/CI2.yml

```diff
@@ -127,5 +127,10 @@ jobs:
           cmake --build build --parallel 2
           CTEST_OUTPUT_ON_FAILURE=1 cmake --build build --target test
           cmake --install build
+      - name: Format Cabana
```

Comment:
Curious if this will find clang format 10

Reply:
We're getting the default: https://github.com/kokkos/ci-containers/blob/071c72d137a9fa8f203dc5bc2b4572be8cc210cc/ubuntu#L8 

Which looks to be working